src/main/java/iterator/Extra.java

```diff
@@ -50,7 +58,9 @@ public static <A, B> RichIterator<Pair<A, B>> product(Iterable<A> a, Iterable<B>
      * @implNote try implement it with only 2 lines of code !
      */
     public static RichIterator<Integer> multiplicationBoard() {
-        throw new NotImplementedException();
+        Iterable<Integer> line = iterate(1, x -> x + 1).take(10).toList();
```

Comment:
Converting to list is a bit over complexity because it implements a lot of unused logic.
```suggestion
        Iterable<Integer> line = () -> iterate(1, x -> x + 1).take(10);
```